Add Rake Task To Update All Local Authority Links

AlanGabbianelli opened the pull request:
We have bad data:
```
irb(main):006:0> pp Link.distinct.pluck(:status)
[nil,
 "broken",
 "Connection failed",
 "missing",
 "500",
 "200",
 "ok",
 "Too many redirects",
 "503",
 "Timeout Error",
 "caution",
 "410",
 "404",
 "403"]
```
This should be:
```
irb(main):001:0> pp Link.distinct.pluck(:status)
["broken", "caution", "missing", "ok"]
```

This rake task will run through all the `Links` and change the `status`
to one of those four.

thomasleese commented on lib/tasks/check-links/link_checker.rake:
```diff
@@ -27,9 +27,17 @@ task "check-links": :environment do
+
+  desc "Check links & update link status for all local authorities"
+  # This will run in the background as a queue of Sidekiq jobs.
+  # The jobs will take a long time to complete, even a few hours.
```
If you include this comment in the description string, it'll be shown to users if they ask for help on the task which might be useful.

Commit:
Add Rake Task To Update All Local Authority Links

We have bad data:
```
irb(main):006:0> pp Link.distinct.pluck(:status)
[nil,
 "broken",
 "Connection failed",
 "missing",
 "500",
 "200",
 "ok",
 "Too many redirects",
 "503",
 "Timeout Error",
 "caution",
 "410",
 "404",
 "403"]
```
This should be:
```
irb(main):001:0> pp Link.distinct.pluck(:status)
["broken", "caution", "missing", "ok"]
```

This rake task will run through all the `Links` and change the `status`
to one of those four.

edwardkerry reviewed (approved):
Even better!
